libs/liquidity/src/lib/liquidity-table.spec.tsx

```diff
@@ -41,16 +43,16 @@ describe('LiquidityTable', () => {
       h.querySelector('[ref="eText"]')?.textContent?.trim()
     );
     const expectedHeaders = [
+      undefined,
```

Comment:
Can you add a comment about why this is undefined. Otherwise its a bit odd/confusing